pom.xml

```diff
@@ -39,17 +39,15 @@
 
   <properties>
     <buildnumber.maven.plugin.version>1.4</buildnumber.maven.plugin.version>
-    <cobertura.version>2.1.1</cobertura.version>
```

Comment:
Can you put this back, I mean to just remove it from the reporting section. I think having the normal plugin in the pluginManagement section is fine, if you don't want to use it downstream then you don't have to. If we remove it entirely from here we'll have to change quite a few downstream projects.

Reply:
done, thanks